altlas/Assets/scripts/LeverScript.cs

```diff
@@ -4,9 +4,16 @@
 
 public class LeverScript : MonoBehaviour, ClickableInterface {
   public GlobeMovingScript globeMovingScript;
+  public RotateGlobeToMapPositionScript rotateScript;
+  string geoCords = "E 68°53'00\"-E 90°52'00\"/N 34°22'00\"-N 05°00'00\"";
```

Comment:
This is probably a bigger change, but I would have passed the `float[]` to `rotateGlobe` instead of a string, instead of parsing such an awkward string inside the function.

Reply:
(a `Coordinates` object would have been most convenient to use)